application/views/EnrollmentForm.php

```diff
@@ -60,7 +60,7 @@
           </div>
           <div id="div-page3" class="div-block page3">
             <div class="div-name-email">
-              <div class="div div-name">
+              <div class="div div-name" style="display: none">
                 <p class="p p-firstname">Your first and last name: <span class="asterisk">*</span></p><input type="text" class="entry w-input" maxlength="256" name="name" data-name="Field" placeholder="Enter your first name" id="field" required="">
               </div>
```

Comment:
Let's just get rid of this if we aren't going to use this anymore

Reply:
Just saying remove that field caused an unknown error regarding the enter button, I will do another test on this around Wednesday

Reply:
@jameszu I've found the solution. Hit me up and we can do some peer programming for the rest of this PR.